Building-with-IntelliJ-IDEA.md

```diff
@@ -52,64 +50,86 @@ When first viewing the project in IntelliJ IDEA you may come across this error:
+![successful_build.png](img%2Fbuilding-with-intellij%2Fsuccessful_build.png)
 
-Now, we need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). So, still in same Maven window, just click the Refresh icon:
+We now need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). `Generate Sources and Update folders for all projects`
 
-![project-maven-refresh](https://i.imgur.com/L2SmWtF.png)
+![generate_sources.png](img%2Fbuilding-with-intellij%2Fgenerate_sources.png)
 
 ## Running the project
 
-Now locate *Project* in sidebar and click that open project view, and expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. And all you need to do now is right-click `RuneLite` class and select run, and you are done, RuneLite should open:
+Now locate `Project` in sidebar and click that open project view, and expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. 
+And all you need to do now is right-click the `RuneLite` class and select `run ‘RuneLite.main()’`
 
-![project-view-run-main](https://i.imgur.com/RXkMc48.png)
+![run_runelite.png](img%2Fbuilding-with-intellij%2Frun_runelite.png)
+
+And you are done! RuneLite should open.
 
 If you are having any issues with this step see [Troubleshooting](#troubleshooting) section.
 
 ## Conclusion
 
-Success! You can switch between running the client and building the Maven project by switching between the **RuneLite** configuration and your custom Maven configuration (should be named *runelite-parent [install]* by default):
+Success! You can now rebuild RuneLite by pressing the green play button near the top right corner of IntelliJ.
 
-![select-run-config](https://i.imgur.com/zKSqojU.png)
+![quick_run_runelite.png](img%2Fbuilding-with-intellij%2Fquick_run_runelite.png)
 
 # Troubleshooting
 
 ## Missing git
 
 If you are getting error about `git.exe` (or `git` on linux and mac) missing, you will need to first download and install Git for your OS. Git is version control software and implementation that RuneLite uses to store and track history of it's source code. To download git, just [go here and select your OS version](https://git-scm.com/downloads).
 
-## Missing JDK
-
-If you are getting errors about missing JDK, make sure to install JDK 11 (see start of this page) and then navigate to *File* -> *Project Structure* like this:
-
-![missing-jdk-project-structure](https://i.imgur.com/IzAKzOH.png)
+## RuneLite fails to build
 
-Then press *+* button and *Add JDK* and locate directory of your installed JDK:
+ Make sure that you don't have any changes in the code and make sure to pull the lastest changes from master. Try restarting the editor. At this point, if you've followed all instructions and [Skipping tests](#optional-skipping-tests) doesn't work, ask for help in the [Runelite Discord](https://discord.gg/runelite)
```

Comment:
putting this above other troubleshooting entries feels like it defeats the purpose of those troubleshooting entries